admin-dev/themes/new-theme/js/pages/product/components/categories/index.js

```diff
@@ -99,30 +95,17 @@ export default class CategoriesManager {
       this.toggleAll(false);
     });
 
+    //@todo: where to store these ids? (should be able to get on page load, but still need to store them because everytime modal is opened they will change, but will not be saved to serverside)
+    const productCategoryIds = [2, 3];
```

Comment:
As I explained above these IDs are never stored, they are fetched right on time when the button is clicked by "parsing" the hidden inputs in the form (present in that "tags")